src/AcceptanceTests/Infrastructure/DefaultServer.cs

```diff
@@ -38,6 +38,7 @@ public Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescriptor,
             var recoverability = configuration.Recoverability();
             recoverability.Delayed(delayed => delayed.NumberOfRetries(0));
             recoverability.Immediate(immediate => immediate.NumberOfRetries(0));
+            configuration.SendFailedMessagesTo("error");
 
             var transportConfig = configuration.UseTransport<MsmqTransport>();
```

Comment:
I'd say just switch the config , if still all green then do it, if not raise a separate issue to track?